codeforboston/maple
# Delete own testimony

# Summary

closes #1382

**1) the ability to self-report was not being displayed; this restriction has been lifted**

**2) the self-report modal itself has been updated to reflect the changes requested in issue #1382**

# Checklist

  - No shared components used

# Screenshots

### **1) enabling self-report**

in the pictures below, the account "Merritt Baggett" is the example self-reporting user

Before | After:

![reporting](https://github.com/codeforboston/maple/assets/73559781/67e2b460-fde3-4561-a8e5-29f782b4217a) ![image](https://github.com/codeforboston/maple/assets/73559781/afe0fae5-2ebc-46b1-8c1d-a1bf41358b37)

### **2) updated Modal text and options**

![image](https://github.com/codeforboston/maple/assets/73559781/ad867097-a81d-4072-a1dc-6d3c0a3f20a3)

# Known issues

No known issues at time of PR submission

# Steps to test/reproduce

1. Log into MAPLE with a personal test account
2. go to a bill and create a testimony if one doesn't already have testimony authored on their test account
3. go to either the relevant bill page or the testimony tab on View Profile and make sure the Report option is displayed for your test account
4. click "Report" and make sure the Self-Report version of the modal is displayed and not the standard version.
5. close the modal and click "More Details" for testimony authored by your account
6. find "Report Testimony" on the right side block column and click that
7. make sure the Self-Report version of the modal is displayed and not the standard version.
